core/src/main/java/be/isach/ultracosmetics/cosmetics/type/EmoteType.java

```diff
@@ -182,6 +182,66 @@ public static void checkEnabled() {
 
         /* WINK END */
 
+        /* RAISE EYEBROWS BEGIN */
+        EmoteType RAISE_EYEBROWS = new EmoteType("RaiseEyebrows", 3);
+        RAISE_EYEBROWS.appendTexture("b69b3af7f829d1d9c18dd9c94c79cef49c54bffa16d4c334c20e4033baf244b7");
+        RAISE_EYEBROWS.appendTexture("75ce894bf9a1552810afd4a8419e7ec362dab414d88ae287f2a5a1b65033fc13");
+        RAISE_EYEBROWS.appendTexture("b69b3af7f829d1d9c18dd9c94c79cef49c54bffa16d4c334c20e4033baf244b7");
+        RAISE_EYEBROWS.appendTexture("75ce894bf9a1552810afd4a8419e7ec362dab414d88ae287f2a5a1b65033fc13");
+        RAISE_EYEBROWS.appendTexture("b69b3af7f829d1d9c18dd9c94c79cef49c54bffa16d4c334c20e4033baf244b7");
+        RAISE_EYEBROWS.appendTexture("75ce894bf9a1552810afd4a8419e7ec362dab414d88ae287f2a5a1b65033fc13");
+        RAISE_EYEBROWS.appendTexture("b69b3af7f829d1d9c18dd9c94c79cef49c54bffa16d4c334c20e4033baf244b7");
+        RAISE_EYEBROWS.appendTexture("75ce894bf9a1552810afd4a8419e7ec362dab414d88ae287f2a5a1b65033fc13");
+        RAISE_EYEBROWS.appendTexture("b69b3af7f829d1d9c18dd9c94c79cef49c54bffa16d4c334c20e4033baf244b7");
+        RAISE_EYEBROWS.appendTexture("75ce894bf9a1552810afd4a8419e7ec362dab414d88ae287f2a5a1b65033fc13");
+
+        /* RAISE EYEBROWS END */
+
+        /* SCARY CANDLE BEGIN */
+        EmoteType SCARY_CANDLE = new EmoteType("ScaryCandle", 5);
+        SCARY_CANDLE.appendTexture("8c54055ee8f9e7638830c79ff12de81e9c7c4e8f3ba3db555019a65872b9c6c");
+        SCARY_CANDLE.appendTexture("2efb5c198fe74feb988dcb9af890b69e8169f5714a7ab4fe8795bd1f66df637f");
+        SCARY_CANDLE.appendTexture("8c54055ee8f9e7638830c79ff12de81e9c7c4e8f3ba3db555019a65872b9c6c");
+        SCARY_CANDLE.appendTexture("2efb5c198fe74feb988dcb9af890b69e8169f5714a7ab4fe8795bd1f66df637f");
+        SCARY_CANDLE.appendTexture("8c54055ee8f9e7638830c79ff12de81e9c7c4e8f3ba3db555019a65872b9c6c");
+        SCARY_CANDLE.appendTexture("2efb5c198fe74feb988dcb9af890b69e8169f5714a7ab4fe8795bd1f66df637f");
+        SCARY_CANDLE.appendTexture("8c54055ee8f9e7638830c79ff12de81e9c7c4e8f3ba3db555019a65872b9c6c");
+        SCARY_CANDLE.appendTexture("2efb5c198fe74feb988dcb9af890b69e8169f5714a7ab4fe8795bd1f66df637f");
+        SCARY_CANDLE.appendTexture("8c54055ee8f9e7638830c79ff12de81e9c7c4e8f3ba3db555019a65872b9c6c");
+        SCARY_CANDLE.appendTexture("2efb5c198fe74feb988dcb9af890b69e8169f5714a7ab4fe8795bd1f66df637f");

```

Comment:
They are not redundant frames, they are part of the animation. Without them this would happen:

1. Frame 1
2. Frame 2
3. Long pause because of: 
https://github.com/datatags/UltraCosmetics/blob/ce4cd751e5f145fcb404287bec71069bb529eca2/core/src/main/java/be/isach/ultracosmetics/cosmetics/emotes/EmoteAnimation.java#L13
4. (replay animation)

Did you test it in-game? The whole emote is much slower this way. Are you sure that this looks good? I tested it in-game and i don't like it.

Reply:
Fortunately you didn't commit those changes (I didn't enable "Allow edits by maintainers"). This is why code reviews are so important. 😉

Reply:
Ah ok, looks good!